src/cript/nodes/primary_nodes/primary_base_node.py

```diff
@@ -1,36 +1,79 @@
+import uuid
 from abc import ABC
 from dataclasses import dataclass, replace
 
 from cript.nodes.core import BaseNode
-from cript.nodes.supporting_nodes.user import User
 
 
-class PrimaryBaseNode(BaseNode, ABC):
+def get_uuid_from_uid(uid):
+    return str(uuid.UUID(uid[2:]))
+
+
+class UUIDBaseNode(BaseNode):
```

Comment:
I think if other nodes will inherit from this it might be a better idea to put it somewhere outside of `primary_base_node` because this implies that this base is only for nodes found within the `primary_nodes` directory. It doesn't make sense for any subobjects to inherit from a primary base node

Reply:
done

Reply:
I honestly, agree, having one class per file makes it easier to read